pkg/web/openapi/swagger-ui/api/v1/api.swagger.json

```diff
@@ -2,14 +2,14 @@
   "swagger": "2.0",
   "info": {
     "title": "Conduit REST API",
-    "version": "0.1",
+    "version": "v0.1.0",
     "contact": {
       "name": "Conduit project",
       "url": "https://github.com/conduitio/conduit"
     },
     "license": {
       "name": "Apache License Version 2.0",
-      "url": "https://github.com/ConduitIO/conduit/blob/main/README.md"
+      "url": "https://github.com/ConduitIO/conduit/blob/main/LICENSE.md"
```

Comment:
I swear I updated these ???? 🤦

Reply:
You did in the proto file, but seems like you didn't generate the code 🙈